src/main/java/com/netgrif/application/engine/petrinet/web/PetriNetController.java

```diff
@@ -114,6 +122,32 @@ public PetriNetReferenceResources getAll(@RequestParam(value = "indentifier", re
         }
     }
 
+    @Authorizations(value = {
+            @Authorize(authority = "PROCESS_VIEW_MY")
+    })
+    @ApiOperation(value = "Get all processes", authorizations = @Authorization("BasicAuth"))
+    @RequestMapping(value = "/my", method = GET, produces = MediaTypes.HAL_JSON_VALUE)
+    public PetriNetReferenceResources getMy(@RequestParam(value = "indentifier", required = false) String identifier, @RequestParam(value = "version", required = false) String version, Authentication auth, Locale locale) {
+        LoggedUser user = (LoggedUser) auth.getPrincipal();
+        List<PetriNetReference> references;
+        if (identifier != null && version == null) {
+            references = service.getReferencesByIdentifier(identifier, user, locale).stream().filter(ref -> ref.getAuthor().getId().equals(user.getId())).collect(Collectors.toList());
```

Comment:
1. logic should not be implemented in a controller
2. filtering should be done in the database

Reply:
This is how was implemented in other methods, I just copied the existing implementation and modified to fullfill the method meening.